dashboard/test/ui/features/instructions/csp_top_instructions_eyes.feature

```diff
@@ -8,12 +8,20 @@ Scenario: CSD and CSP Top Instructions
   And I wait for the page to fully load
   Then I see no difference for "teacher in applab level with rubric"
   Then I click selector ".uitest-feedback"
+  And I wait for the page to fully load
   Then I see no difference for "teacher in applab level viewing rubric"
   Then I click selector ".uitest-instructionsTab"
+  And I wait for the page to fully load
```

Comment:
unless you are triggering a full page navigation/reload, I don't think these statements will do anything. unless a reload is happening, please remove this one and the one you've added above